multiqc/utils/search_patterns.yaml

```diff
@@ -80,6 +80,18 @@ bcl2fastq:
   fn: "Stats.json"
   contents: "DemuxResults"
   num_lines: 300
+bclconvert/runinfo:
+  fn: "RunInfo.xml"
+  contents: "<?xml"
```

Comment:
@andrei-seleznev: Post-merge comment - I'm going to remove these `contents:` search criteria. Partly because the (unescaped) tag here breaks the MultiQC documentation rendering 😆 But also the filenames are unique enough that they shouldn't need any additional search criteria. Also looking into the file contents will slow down the file search a little.

Let me know if you know of other tools making files with identical filenames, where the `contents` search criteria here would be needed to disambiguate the outputs 👍🏻